test/framework/easyblock.py

```diff
@@ -1025,10 +1025,16 @@ def test_init_extensions(self):
 
         test_ec = os.path.join(self.test_prefix, 'test.eb')
         test_ec_txt = toy_ec_txt.replace("('barbar', '0.0', {", "('barbar', '0.0', {'easyblock': 'DummyExtension',")
+        test_ec_txt = test_ec_txt.replace("('barbar', '0.0', {", "('barbar', '0.0', {'maxparallel': 3,")
+        test_ec_txt = test_ec_txt.replace("('bar', '0.0', {", "('bar', '0.0', {'maxparallel': 1,")
+        test_ec_txt += "\nparallel = 5"
         write_file(test_ec, test_ec_txt)
         ec = process_easyconfig(test_ec)[0]
         eb = get_easyblock_instance(ec)
 
+        # require to trigger setting of 'parallel' value
```

Comment:
"required"

Reply:
If you move that setting of orig-parallel to the ctor this is not really required anymore, unless you add a top-level `maxparallel` and test the mentioned case of an extension having a higher value for it and it should still work.

E.g. instead of the EC `parallel` (why do we have that anyway? Isn't that basically the same as `maxparallel`?) use a `maxparallel` of 2 and a build option parallel of 5, then barbar should still yield in a parallel value of 3